### ISSUE TITLE
I linked my account but it doesnt show anything

### ISSUE THREAD

**Issue by GerardPolloRebozado** [guerrerotook/securitas-direct-new-api#145]
![image](https://github.com/guerrerotook/securitas-direct-new-api/assets/37554513/1467bf2b-7354-4238-877f-1f2beb55efaa)
Also in the integrations panel shows this error:
![image](https://github.com/guerrerotook/securitas-direct-new-api/assets/37554513/4997da68-8be7-47de-8861-d2999b8ff8c1)

**Comment by mgomezgu** [comment]
It happens exactly the same as you. Have you been able to solve it? I have a question about the first "code" that asks for the integration to which it refers? (This is not the mistake)
![Screenshot_2023-08-28-11-21-36-521_io homeassistant companion android-edit](https://github.com/guerrerotook/securitas-direct-new-api/assets/129875800/cd701a26-3f21-4580-8ead-93d49eed017b)

**Comment by mgomezgu** [comment]
This os t'he log.


2023-08-28 11:37:48.097 DEBUG (MainThread) [custom_components.securitas] Log in Securitas: True
2023-08-28 11:37:48.098 DEBUG (MainThread) [custom_components.securitas.securitas_direct_new_api.apimanager] Making request with device_id XXXXXXXXXXXX, uuid XXXXXXXXXXXX and idDeviceIndigitall XXXXXXXXX
2023-08-28 11:37:49.135 DEBUG (MainThread) [custom_components.securitas.securitas_direct_new_api.apimanager] --------------Response--------------
2023-08-28 11:37:49.135 DEBUG (MainThread) [custom_components.securitas.securitas_direct_new_api.apimanager] {"errors":[{"message":"Unauthorized","name":"SecurityError","time_thrown":"2023-08-28T09:37:49.073Z","data":{"auth-code":"10010"},"path":["xSInstallations"]}],"data":{"xSInstallations":null}}

2023-08-28 11:37:49.135 ERROR (MainThread) [custom_components.securitas.securitas_direct_new_api.apimanager] Unauthorized
2023-08-28 11:37:54.896 WARNING (SyncWorker_1) [homeassistant.helpers.service] Unable to find services.yaml for the securitas integration

**Comment by mgomezgu** [comment]
I have managed to make it work. I have created a second user, then I have entered this user in the mobile app (it asks for a password change) and once it works on the mobile, I have configured the homeassistant.

**Comment by GerardPolloRebozado** [comment]
> It happens exactly the same as you. Have you been able to solve it? I have a question about the first "code" that asks for the integration to which it refers? (This is not the mistake) ![Screenshot_2023-08-28-11-21-36-521_io homeassistant companion android-edit](https://user-images.githubusercontent.com/129875800/263659146-cd701a26-3f21-4580-8ead-93d49eed017b.jpg)

I dont know what the code is but checking the github saw that on an update the code was optional so I typed 0 and then in the config removed the value. Do you know what the code is?

**Comment by SirLouen** [comment]
> I have managed to make it work. I have created a second user, then I have entered this user in the mobile app (it asks for a password change) and once it works on the mobile, I have configured the homeassistant.

Do you have more than 1 installation under the other user?

**Comment by GerardPolloRebozado** [comment]
> > I have managed to make it work. I have created a second user, then I have entered this user in the mobile app (it asks for a password change) and once it works on the mobile, I have configured the homeassistant.
> 
> Do you have more than 1 installation under the other user?

I only have 1 installation. Do I need to sign in with mi admin user or a second user?

**Comment by mgomezgu** [comment]
I only have one installation. But it was impossible for me to connect it with the main user, or that's why I made a second user.

**Comment by SirLouen** [comment]
> I only have 1 installation. Do I need to sign in with mi admin user or a second user?
> I only have one installation. But it was impossible for me to connect it with the main user, or that's why I made a second user.

Ok, the thing is that I found some issues under multiple installations and I was wondering if these affected to you the same. When you create a new user, you create the new user for just one installation (regardless the amount of installations you already have), this is why was partially suspecting that this could be the problem

But it's super weird that you created a second user and it worked. There are a ton of weird things in this integrations because the Securitas API has a ton of things that are almost impossible to check them by the developer (because he is only seeing what his installation return to him). 

So basically without the detailed logs from the connection it's impossible to know what it's hindering the setup.

**Comment by GerardPolloRebozado** [comment]
This is the error: Could not log in to Securitas: Usuario bloqueado. Para desbloquearlo, pulsa sobre olvidé mi contraseña y sigue los pasos.
It just says the user is blocked and I need to reset my password, logged out from the official app and can login again perfectly. 
tried changing the password and still same issue.

**Comment by guerrerotook** [comment]
Sorry for the inconvenience. The code that you are referring on the screenshots is for Home Assistant to required a code to arm and disarm, is not related to Securitas Direct. 

Also, you can enable debug logs in the integration so see what is happening exactly. 


```yaml
logger:
  logs:
    custom_components.securitas_direct: debug
```

**Comment by zav13r** [comment]
I recently had a similar issue. After speaking with Securitas customer support, they confirmed that a second user is required to prevent an account being locked.

If you use the same login for mobile app and this integration, you will surely encounter issues because you may only be logged into one place at a time.

So, I have the integration connected through my admin securitas account, and my phone logged in using a second user.

Before this, I received the same error messages above and HA showed the integration, but no devices. Adding a second user, unlocking my account (Securitas customer support) and then setting up the integration again solved the problem for me.
